avalon/tools/loader/widgets.py

```diff
@@ -1295,7 +1298,18 @@ def _load_representations_by_loader(loader, repre_ids, dbcon,
     return error_info
 
 
-def _load_subsets_by_loader(loader, subset_ids, dbcon, options):
+def _load_subsets_by_loader(loader, subset_ids, dbcon, options,
+                            subset_version_docs=None):
```

Comment:
Seems like `subset_version_docs` is required (and important) argument.

Reply:
I Agree that it should be documented as optional. @kalisp